pkg/apply/applier.go

```diff
@@ -145,6 +143,9 @@ func (a *Applier) Run(ctx context.Context, invInfo inventory.InventoryInfo, obje
 			InfoHelper:   a.infoHelper,
 			Mapper:       mapper,
 			InvClient:    a.invClient,
+			Destroy:      false,
+			// TODO(seans): Remove this once Filter interface implemented.
```

Comment:
The `LocalNamespaces` field has been removed in favor of the LocalNamespacesFilter.